### ISSUE TITLE
Network input get stuck 

### ISSUE THREAD

**Issue by zabolotiny** [Sinclair81/Homebridge-Logo-TS#11]
Quite often I can see that networkinput stays 1. And changes in Logo config is not going to help because it Reset will stuck you won't be able to toggle relay contact.
Is there any solution for that? Maybe some stable configuration which you can suggest?

**Comment by zabolotiny** [comment]
https://user-images.githubusercontent.com/10601271/132983030-bb261d5f-b79b-4dce-baaa-499f562b9f7d.mov

**Comment by zabolotiny** [comment]
Some more info here:
When that happens I receive an error in log for specific switch:
`>> DBWrite failed. Code #589856 -  ISO : An error occurred during send TCP : Other Socket error (32)`
What is that about?

**Comment by zabolotiny** [comment]
<img width="808" alt="image" src="https://user-images.githubusercontent.com/10601271/132983590-d781daa1-3d57-48bf-acc0-8d9a8cbaea4a.png">
@Sinclair81 Maybe it makes sense to add some retry mechanism here with retry count?

**Comment by zabolotiny** [comment]
Please check [https://github.com/Sinclair81/Homebridge-Logo-TS/pull/12](url) if that looks ok for you?

**Comment by acresp** [comment]
I tried the update but the scenes are not working for me, same thing: only 1 accessory which turns on.
My LOGOs are 0BA8.

**Comment by Sinclair81** [comment]
Yes, that should also be done there.
But I don't have time for that because it's more important to me to get the new HomeKit Logo Bridge up and running.
If one of you makes the changes and a PR for it I will test it and publish it as soon as possible.

**Comment by acresp** [comment]
> Yes, that should also be done there.
> But I don't have time for that because it's more important to me to get the new HomeKit Logo Bridge up and running.
> If one of you makes the changes and a PR for it I will test it and publish it as soon as possible.

As discussed I prefer your ESP32 bridge solution. I guess that you have more flexibility in the programmation and I think It can be a more reliable solution (but TBH the homebridge solution, for using it every day, it's very strong! My setup is backuped by a UPS and running in a docker image on a Synology server, for more than 1 year I have never had any issue!).

**Comment by acresp** [comment]
Talking for the Modbus part, there's something that don't work correctly in the last release.
Today my daugther told Siri to light up the Kitchen. I was surprised because all lights accessories of the Kitchen were powered on. Good, the "scene" trick is working, as more than 1 accessory was turned on. 
But then, if we try to turn off one of the accessory previously powered by this "scene", they turn on again directly. Is there something staying in the loop somewhere ? 🙂
TBH it's not a scene, I was surprised, it's just Siri which is grouping multiple accessories which are in the same room, if the demand is related to a particular room. So it's becoming a kind of scene... lol.

**Comment by Sinclair81** [comment]
OK, I thought I solved this problem with the last update.
I looked in the Siemens forum and actually found a solution.
[V_pulse from ixo65](https://support.industry.siemens.com/tf/ww/en/posts/how-to-reset-network-input-in-logo-8-fbd-program/142779/?page=0&pageSize=10#post571778)
<img width="988" alt="Bildschirmfoto 2021-09-29 um 16 54 28" src="https://user-images.githubusercontent.com/21153794/135297299-cf4c218f-ed61-4d73-b585-7cb9de81163e.png">
I created a UDF from it. You only have to connect the existing network input to it.
And then has an output for whatever is to be controlled and another output for a network output which must have exactly the same memory address as the network input.
Since 64 network inputs, network outputs and UDF instances (16 UDF types) are available, this should be feasible!
